build_tools/cmake/iree_copts.cmake

```diff
@@ -12,8 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#-------------------------------------------------------------------------------
+# Abseil configuration
+#-------------------------------------------------------------------------------
+
 include(AbseilConfigureCopts)
 
+# By default Abseil strips string literals on mobile platforms, which means
+# we cannot run IREE binaries via command-line with proper options. Turn off
+# the stripping.
+# TODO: we might still want to strip when compiling IREE into Android Java apps.
```

Comment:
Yeah, library code hosted within an application will want special logic for interfacing with C++ flags, if we support flags at all. We should be able to use purely C API parameters and build defines for most uses.

I worked on some flags forwarding code for C++ flag definitions -> iOS+Android apps -> Java/objc app flags and shared C++ flags before. Could dig that code up if we really want it.

Reply:
+1. I think we can defer this to a later time.